plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/data/ExecuteInsertBatchImpl.java

```diff
@@ -143,9 +148,11 @@ StringBuilder prepareQueryForStatement(
             return query;
         }
         boolean hasKey = false;
+        List<Integer> skipAttributes = new ArrayList<Integer>();
```

Comment:
Why can't we have a list of attributes themselves, not just their indexes?

Update: I see why. Then let's inverse the meaning of this list by having _used_ attributes instead of _skipped_.